src/fides/api/ops/service/connectors/fides/fides_client.py

```diff
@@ -145,7 +146,11 @@ def create_privacy_request(
 
     @sync
     async def poll_for_request_completion(
-        self, privacy_request_id: str, timeout: int, interval: int
+        self,
+        privacy_request_id: str,
+        timeout: int,
+        interval: int,
+        async_client: AsyncClient | None = None,
```

Comment:
Will this work in older versions of Python?

Reply:
It works on 3.7+ with `from __future__ import annotations`

Reply:
While we've largely been using the older format, folks have been starting to use this newer format in places for awhile now too.